code/game/machinery/hologram.dm

```diff
@@ -137,10 +137,6 @@ Possible to do for anyone motivated enough:
 
 	return ..()
 
-/obj/machinery/holopad/AltClick(mob/living/carbon/human/user)
-	if(isAI(user))

```

Comment:
if mob/living/carbon/human/user isAI. nice meme

This should've been handled by AIAltClick anyway.